jinahub/encoders/audio/AudioCLIPEncoder/audio_clip/model/clip/clip.py

```diff
@@ -158,6 +158,9 @@ def patch_float(module):
 
         model.float()
 
+    global _tokenizer
```

Comment:
why do we touch this? let's keep it as it was

Reply:
@tadejsv